translate.go

```diff
@@ -1137,20 +1111,14 @@ func (r *translateFileReader) read(p []byte) (n int, err error) {
 		return 0, nil
 	}
 
-	if max := sz - r.offset; max > int64(len(p)) {
-		// If p is not large enough to hold a single entry,
-		// return an error so the client can increase the
-		// size of p and try again.
-		return 0, ErrTranslateReadTargetUndersized
-	} else if int64(len(p)) > max {
+	if max := sz - r.offset; int64(len(p)) > max {
 		// Shorten buffer to maximum read size.
 		p = p[:max]
 	}
 
 	// Read data from file at offset.
 	// Limit the number of bytes read to only whole entries.
```

Comment:
I guess this comment needs to change now?

Reply:
...and potentially the help on this method since we might not get an integer number of valid entries.